scripts/import_data.py

```diff
@@ -0,0 +1,420 @@
+                        help="Column names that will be made private")
+    parser.add_argument("-s", "--skip_fields", type=str, nargs='+', default=[],
+                        help="Column names that will be skipped")
+    parser.add_argument("-c", "--collection", type=str, required=True,
+                        help="Name of the collection to import data into. "
+                             "A new collection will be created if it does not exist.")
+    parser.add_argument("-u", "--user", type=str,
+                        help="Koillection user that will become the owner of the newly created collection "
+                             "(not required when there is only one user)")
+
+    parser.add_argument("--skip_empty_columns", action="store_true",
+                        help="Removes all columns that don't have a value in any item")
+    parser.add_argument("--skip_empty_fields", action="store_true",
+                        help="Removes fields that don't have a value (works per-item)")
+
+    parser.add_argument("--host", type=str, help="Database host ip")
+
+    args = vars(parser.parse_args())
+
+    args["private_fields"] = [field_name.lower() for field_name in args["private_fields"]]
+    args["skip_fields"] = [field_name.lower() for field_name in args["skip_fields"]]
+
+    return args
+
+
+def load_environment_variables(compose_file: str) -> dict:
+    """
+    Loads environmental variables from the koillection service, from the provided compose file
+    :param compose_file: path to the compose file
+    :return: dictionary containing the environmental variables and their values
+    """
+
+    with open(compose_file, 'r') as file:
+        compose = yaml.safe_load(file)
+
+    environment = {}
+    for item in compose["services"]["koillection"]["environment"]:
+        key, value = item.split('=')
+        environment[key] = value
+
+    return environment
+
+
+def main() -> None:
+    """The main importer function. Loads all the required data and imports items into the database"""
+
+    args = parse_args()
+    environment = load_environment_variables(args["compose_file"])
+
+    cnx = mysql.connector.connect(
+        user=environment["DB_USER"],
+        password=environment["DB_PASSWORD"],
+        host=args["host"] if args["host"] is not None else get_database_host(environment["DB_HOST"]),
+        port=environment["DB_PORT"],
+        database="koillection"
+    )
+    cursor = cnx.cursor()
+
+    items, headers = load_data(args["file"], args["sheet"], args["skip_empty_columns"], args["skip_empty_fields"])
+
+    username = args["user"] if args["user"] else db_get_username(cursor)
+
+    cursor.execute(f"SELECT id FROM koi_user WHERE username='{username}'")
+    owner_id = cursor.fetchone()[0]
+
+    collection_id = db_get_collection_id(cursor, args["collection"], owner_id)
+
+    # insert all items
+    for item in items:
+        item_id = insert_item(cursor, owner_id, collection_id, item[args["name_column"]])
```

Comment:
try catch
Traceback (most recent call last):
  File "/home/wpanfil/Documents/koillectionPG/scripts/./import_data.py", line 420, in <module>
    main()
  File "/home/wpanfil/Documents/koillectionPG/scripts/./import_data.py", line 397, in main
    item_id = insert_item(cursor, owner_id, collection_id, item[args["name_column"]])
    
and inform user that he either needs to pass or passed wrong val

Please consider